source/frontend/src/features/properties/map/acquisition/detail/AcquisitionFileTabs.tsx

```diff
@@ -114,7 +114,10 @@ export const AcquisitionFileTabs: React.FC<IAcquisitionFileTabsProps> = ({
       tabViews={tabViews}
       defaultTabKey={defaultTab}
       activeTab={activeTab}
-      setActiveTab={setActiveTab}
+      setActiveTab={(tab: FileTabNames) => {
```

Comment:
Just to note that we need to refactor the ViewSelector. It has grown in complexity before the initial implementation.

Reply:
Agreed, may want to just use routing so that user's can link other's to specific tabs and the back button would work.